component/Alerts/styles.css

```diff
@@ -0,0 +1,96 @@
+
+@import url('/styles/nav.css');
+@import url('/styles/styles-global/root.css');
+@import url('/styles.css');
+
+:root{
+    --alert-success-color: #b7e1b4;
+    --alert-info-primary-color: #9999ff;
+    --alert-info-secondary-color: #ffdec5;
+    --alert-error-color: #f8c1c1;
+    --alert-warning-color: #f8f6c4;
+}
+
+.alert-body{
+    font-family: var(--font-family-content);
+    grid-area: main;
+    margin: 0rem 18rem;
+    background-color: var(--white-color);
+    padding: 2rem 2rem;
```

Comment:
If padding is same for all 4 sides, there is another shorthand padding : 2rem, gives the same effect.

Reply:
yes, thanks i will change the size of the avatars and add a close button in alerts.